carbon-design-system/carbon
# fix(ExpandableTile): resolves VO issues

Closes #3375 

This PR updates ExpandableTile so that the screenreader announces the above the fold content and the below the fold content and the state of the tile, expanded or not. 

#### Changelog

**New**

- tileContentID for the aria-labelledby

**Changed**

- `uid` to the `setupGetInstanceId`
- changes the container `div` to an `aside` following what Material does with their tiles because it allowed for the above the fold content to be announced.  
- update tests to verify aria-label on the button and not `ChevronDown16`

**Removed**

- `uid = uid()`, I don't think this was being used anywhere so I deleted it. 
- all the attributes on `ChevronDown16`, by default our icons are hidden so I moved the aria-label to the button that the svg is contained by. 

#### Testing / Reviewing
Verify in Storybook using VoiceOver that this happens: 
- Press Control-Option-Shift-Down Arrow to enter the Web area, VO should announce “Above the fold text”
- Press Control-Option-Right Arrow, VO should announce the button label and the current state of expanded.
- Press the button with Control-Option-Space and VO should announce the expanded state using aria-expanded=“true” before the “Below the fold content” gets focus.
- Press Control-Option-Right Arrow, VO announces should announce “Below the fold content”.
